packages/adapters/eigenda/src/error.rs

```diff
@@ -7,7 +7,7 @@ pub enum Error {
     #[error("authentication failed")]
     AuthenticationFailed,
     #[error("EigenDA client error: {0}")]
-    EigenDAClient(#[from] rust_eigenda_client::errors::EigenClientError),
+    EigenDAClient(#[from] rust_eigenda_v2_client::errors::EigenClientError),
     #[error("other error: {0}")]
     Other(String),
```

Comment:
I noticed we're using the `Other(String)` variant in quite a few places throughout this PR. Might be better to remove this in favour of specific error types - would make error handling more robust and easier to debug.